staging/src/k8s.io/apiserver/pkg/util/flowcontrol/request/config.go

```diff
@@ -20,73 +20,40 @@ import (
 	"time"
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apiserver/pkg/apis/apiserver"
 )
 
 const (
-	minimumSeats                = 1
-	maximumSeats                = 10
-	objectsPerSeat              = 100.0
-	watchesPerSeat              = 10.0
-	enableMutatingWorkEstimator = true
-)
-
-var eventAdditionalDuration = 5 * time.Millisecond
-
-// WorkEstimatorConfig holds work estimator parameters.
-type WorkEstimatorConfig struct {
-	*ListWorkEstimatorConfig     `json:"listWorkEstimatorConfig,omitempty"`
-	*MutatingWorkEstimatorConfig `json:"mutatingWorkEstimatorConfig,omitempty"`
-
-	// MinimumSeats is the minimum number of seats a request must occupy.
-	MinimumSeats uint `json:"minimumSeats,omitempty"`
-	// MaximumSeats is the maximum number of seats a request can occupy
-	//
-	// NOTE: work_estimate_seats_samples metric uses the value of maximumSeats
-	// as the upper bound, so when we change maximumSeats we should also
-	// update the buckets of the metric.
-	MaximumSeats uint `json:"maximumSeats,omitempty"`
-}
+	defaultEnableMutatingWorkEstimator      = true
+	defaultMaximumSeats                uint = 10
+	defaultObjectsPerSeat                   = 100.0
+	defaultWatchesPerSeat                   = 10.0
 
-// ListWorkEstimatorConfig holds work estimator parameters related to list requests.
-type ListWorkEstimatorConfig struct {
-	ObjectsPerSeat float64 `json:"objectsPerSeat,omitempty"`
-}
+	// minimumSeats is the minimum number of seats a request must occupy.
+	minimumSeats = 1
+)
 
-// MutatingWorkEstimatorConfig holds work estimator
-// parameters related to watches of mutating objects.
-type MutatingWorkEstimatorConfig struct {
-	// TODO(wojtekt): Remove it once we tune the algorithm to not fail
-	// scalability tests.
-	Enabled                 bool            `json:"enable,omitempty"`
-	EventAdditionalDuration metav1.Duration `json:"eventAdditionalDurationMs,omitempty"`
-	WatchesPerSeat          float64         `json:"watchesPerSeat,omitempty"`
-}
+var defaultEventAdditionalDuration = 5 * time.Millisecond
```

Comment:
I know this was a `var` before, but I do not understand why; couldn't/shouldn't it be a constant?

Reply:
Cool, I didn't know that a `time.Duration` can be a `const`. :D Changed.